widget/embedded/src/containers/WidgetInfo/WidgetInfo.tsx

```diff
@@ -12,11 +14,26 @@ export const WidgetInfoContext = createContext<
 export function WidgetInfo(props: React.PropsWithChildren) {
   const { manager } = useManager();
   const swaps = getPendingSwaps(manager);
+  const walletsDetails = useWalletsStore.use.connectedWallets();
+  const isWalletLoading = useWalletsStore.use.loading();
+  const totalBalance = calculateWalletUsdValue(walletsDetails);
+  const refetchBalance = useWalletsStore.use.getWalletsDetails();
   const value = useMemo(() => {
```

Comment:
Why it would be inside an Memo, when it will be changed always?

Reply:
It is an Eslint error. I temporarily disable the rule.